corehq/apps/app_manager/suite_xml.py

```diff
@@ -1922,11 +1923,25 @@ def get_datum_meta_module(self, module, use_filter=False):
                     detail_inline = bool(detail.pull_down_tile)
                     break
 
+            fixture_select_filter = ''
+            if datum['module'].fixture_select.active:
+                datums.append({
+                    'datum': SessionDatum(
+                        id='fixture_value',
+                        nodeset="instance('item-list:{ft}')/{ft}_list/{ft}"
+                            .format(ft=datum['module'].fixture_select.fixture_type),
+                        value=datum['module'].fixture_select.variable_column,
+                        detail_select='m{}_fixture_select'.format(datum['module'].id)
+                    )
+                })
+                fixture_select_filter = "[{}]".format(datum['module'].fixture_select.xpath
+                                                      .replace('$fixture_value', "instance('commcaresession')/session/data/fixture_value"))
```

Comment:
I don't think this should be inside the for loop otherwise you'll get a datum added for each existing datum e.g. parent, child, usercase etc.